ts/util/theme.ts

```diff
@@ -1,7 +1,7 @@
 import { ThemeStateType } from '../themes/constants/colors';
 
-export const checkDarkTheme = (theme: ThemeStateType): boolean => theme.includes('dark');
-export const checkLightTheme = (theme: ThemeStateType): boolean => theme.includes('light');
+export const checkDarkTheme = (theme: ThemeStateType): boolean => theme?.includes('dark');
```

Comment:
we only have half a redux store setup on the registration page, but the I18N components needs to know if dark/light theme is enabled currently ( something I want to change, later).
Because of that, the app crashes on the registration page if we don't `?.` it
![image](https://github.com/user-attachments/assets/6843fb35-6d02-4075-8ebe-bc65be269884)

Reply:
Ah okay I see. We might want to abstract the theme into into it's own provider with classic dark as the default values (for onboarding). Happy with the explanation. Resolving